zerver/lib/types.py

```diff
@@ -276,3 +276,8 @@ class RealmPlaygroundDict(TypedDict):
     name: str
     pygments_language: str
     url_prefix: str
+
+
+class MissingMessageDict(TypedDict):
```

Comment:
I think it'd be better to define this type in `soft_deactivation.py`, since it's not a type intended to be used widely.

Reply:
OK. Apply the same change to `UserGroupDict`.